packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.ts

```diff
@@ -346,6 +337,8 @@ export class HorizontalScroll extends FoundationElement {
      * @public
      */
     public scrollToPrevious(): void {
+        this.getRefreshRate();
```

Comment:
Does refresh rate need to be recalculated every time the scrolls happen?

Reply:
This was the only way I could think of to ensure that Firefox returns a value that's as close to the actual refresh rate as it will allow. Chrome and Edge have no issues and always return the correct number.